lib/bigcommerce/connection.rb

```diff
@@ -34,24 +34,24 @@ def api_key=(api_key)
       @configuration[:api_key] = api_key
     end
 
-    def verify_peer=(verify)
+    def verify_ssl=(verify)
```

Comment:
We should keep the original method name for backward compatibility, we should definitely update the readme with this, sorry for the confusion!